package_manifest.yaml

```diff
@@ -549,28 +546,61 @@ plugins_packages:
     rubygem-hammer_cli_foreman_ssh: {}
     rubygem-hammer_cli_foreman_tasks: {}
     rubygem-hammer_cli_foreman_templates: {}
-    rubygem-infoblox: {}
     rubygem-jgrep: {}
     rubygem-jquery-matchheight-rails: {}
-    rubygem-logify: {}
     rubygem-net-ssh-gateway: {}
     rubygem-net-ssh-krb: {}
     rubygem-net-ssh-multi: {}
-    rubygem-newt: {}
     rubygem-opennebula: {}
-    rubygem-openscap: {}
     rubygem-ovirt_provision_plugin: {}
     rubygem-parse-cron: {}
     rubygem-pdf-core: {}
     rubygem-polyglot: {}
     rubygem-prawn: {}
     rubygem-puppetdb_foreman: {}
-    rubygem-radcli: {}
     rubygem-rainbow: {}
-    rubygem-route53: {}
     rubygem-rqrcode: {}
-    rubygem-ruby-hmac: {}
     rubygem-safe_yaml: {}
+    rubygem-smart_proxy_dynflow_core: {}
+    rubygem-systemu: {}
+    rubygem-ttfunk: {}
+    rubygem-wicked: {}
+    rubygem-zscheduler: {}
+
+plugin_nonscl_packages:
+  vars:
+    package_base_dir: 'packages/plugins/'
+    diff_package_skip: false
+    diff_package_tags:
+    - foreman-plugins-nightly-nonscl-rhel7
+    releasers:
+    - koji-foreman-plugins
+    nightly_releaser: koji-foreman-plugins-jenkins
+    repoclosure_config: repoclosure/yum_el7.conf
+    repoclosure_lookaside_repos:
+      - el7-base
+      - el7-updates
+      - el7-epel
+      - el7-extras
+      - el7-puppet-5
+      - el7-foreman-rails-nightly
```

Comment:
Does `nonscl` need these repos? In the other change you didn't include them for nonscl.